app/views/govuk_publishing_components/components/_accordion.html.erb

```diff
@@ -23,10 +23,12 @@
 
   data_attributes ||= {}
   ((data_attributes[:module] ||= "") << " " << "govuk-accordion gem-accordion").strip!
+  ga4_tracking ||= false
+  data_attributes[:module] << " ga4-event-tracker" if ga4_tracking
+  data_attributes[:ga4_expandable] = '' if ga4_tracking
```

Comment:
I think there should be a test in the `.spec` file to check that these attributes are added correctly, along with the individual attributes added for each step further down.

Reply:
Thanks @andysellick - there's now a spec test for the module, `data-ga4-expandable` and `data-ga4-show-all-attributes` and the `data-ga4-event` on the 1 accordion item in the test.